noah/src/main/resources/crawler/promed_mail.py

```diff
@@ -72,15 +54,40 @@ def getEachMail(archNo):
 
 if __name__ == '__main__':
     archNo = set()
-    end_date = datetime.date.today().strftime("%m/%d/%Y")
-    start_date = datetime.datetime(2007, 01, 01).strftime("%m/%d/%Y")
-    filename = "./promedsearchresult.html"
-    request(filename, getNext(start_date, end_date))
-    (listId) = getArchiveNosAndPages(filename)
+    headers = {}
+    headers['User-Agent'] = "Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/51.0.2704.103 Safari/537.36"
+    headers['X-Requested-With'] = 'XMLHttpRequest'
+    headers['Referer'] = "http://promedmail.org/"
+    headers['Accept'] = "application/json, text/javascript, */*"
+    headers['Accept-Encoding'] = "gzip, deflate, sdch"
+    headers['Accept-Language'] = "en-US,en;q=0.8"
+    headers['Connection'] = "keep-alive"
+    headers['Content-Type'] = "application/x-www-form-urlencoded"
+    headers['Host'] = 'promedmail.org'
+    
+    end_date = datetime.date.today()
+    start_date = end_date - datetime.timedelta(7)
+    end_date = end_date.strftime("%m/%d/%Y")
+    start_date = start_date.strftime("%m/%d/%Y")
+    filename = "promedsearchresult.html"
+    dirName = "PromedResult"
```

Comment:
I agree with your point, instead of passing the start and end time, we can pass the `interval` parameter. Good idea!